contrail-agent/hooks/contrail_agent_hooks.py

```diff
@@ -166,6 +173,50 @@ def upgrade_charm():
     utils.update_charm_status()
 
 
+@hooks.hook('nrpe-external-master-relation-changed')
+def nrpe_external_master_relation_changed():
+    update_nrpe_config()
+
+
+def update_nrpe_config():
+    plugins_dir = '/usr/local/lib/nagios/plugins'
+    nrpe_compat = nrpe.NRPE(primary=False)
+    rsync_nrpe_checks(plugins_dir)
+    add_nagios_to_sudoers()
+
+    check_contrail_status_cmd = os.path.join(plugins_dir,
+                                             'check_contrail_status_agent.py')
+    nrpe_compat.add_check(
+        shortname='check_contrail_status_agent',
+        description='Check contrail-status',
+        check_cmd=check_contrail_status_cmd
+    )
+
+    nrpe_compat.write()
+
+
+def rsync_nrpe_checks(plugins_dir):
```

Comment:
i see that this func and next are similar in all files - can you please move them into common_utils.py?

Reply:
All common code is now is `common_utils.py`